lib/std/builtin.zig

```diff
@@ -141,23 +141,49 @@ pub const Mode = enum {
 /// This data structure is used by the Zig language code generation and
 /// therefore must be kept in sync with the compiler implementation.
 pub const CallingConvention = enum {
+    /// This is the default Zig calling convention used when not using `export` on `fn`
+    /// and no other calling convention is specified.
     Unspecified,
+    /// Matches the C ABI for the target.
+    /// This is the default calling convention when using `export` on `fn`
+    /// and no other calling convention is specified.
     C,
+    /// This makes a function not have any function prologue or epilogue,
+    /// making the function uncallable in regular Zig code.
+    /// This can be useful when integrating with assembly.
     Naked,
+    /// Functions with this calling convention are called asynchronously,
```

Comment:
Explaining async with async? Maybe `suspendible stackless coroutines` (with target-specific layout?) as mentioned in https://ziglearn.org/chapter-5/.

Reply:
The point is not to explain what `async` is. I agree some of this is redundant. Amending.